docs/source/models/cause.rst

```diff
@@ -41,29 +41,251 @@ Common data sources for cause models
+
+	For example, the probability that a simulant born during a simulation of cleft lip in the United States 
+	in 2006 is 0.00106, or 0.106%.
+
+Incidence Rates
++++++++++++++++
+
+Incidence rates are defined as the **number of new cases of a condition that occur per person-year of the at-risk 
+population (individuals without condition).** 
+
+	For example, the incidence of multiple sclerosis (MS) in the United States is 2.8 per 100,000 
+	person-years. 
+
+	This suggests that if we followed 100,000 individuals without MS for 1 year each (100,000 people * 
+	1 year = 100,000 person-years), we would expect 2.8 of these individuals to develop MS within this timeframe. 
+
+	Alternatively, if we followed 50,000 individuals without MS for 2 years each (50,000 people * 2 years = 100,000 
+	person-years), we also would expect 2.8 of these individuals to develop MS within this timeframe.
+
+Incidence can be used to **estimate cause model transition rates** and can represent the **probability that a simulant 
+will transition from a susceptible state to an infected state within a given timestep.** 
+
+	For example, with a timestep of one year, the probability that a simulant will transition from a 
+	susceptible (without MS) cause model state to an infected (with MS) cause model state is 2.8*10^(-5).
+
+**A Few Considerations for Incidence Data Sources:**
+
+As mentioned above, the denominator for incidence is person-years of the *at-risk* population, or the population 
+*without* condition (``1 - condition prevalence``). However, in certain scenarios, this may not always be the case.
+
+	When the prevalence of a condition is *small*, ``1 - prevalence`` will *approximately* equal ``1``. In 
+	these cases, incidence may be calculated as the number of new cases per person-years in the *entire* 
+	population as an approximation of a number of new cases per person-years in the *at-risk* population. 
+	This approximation will likely not have a large impact when condition prevalence is low, although the 
+	use and impact of this approximation in incidence data sources used for cause models should be considered.
+
+Further, it is important to consider that cause models are *state*-specific and not necessarily 
+*disease*-specific. What does this mean?
+
+	Consider a cause model in which an individual can trasition from a susceptible state to a mild disease 
+	state OR from a susceptible state to a severe disease state.
+
+	In this case, the incidence rate for overall disease (mild and severe) does not help us estimate the 
+	transition rates from susceptible to mild disease or to severe disease. In these cases, incidence rates 
+	specific to mild and severe disease are needed to inform the specific transitions present in the cause model.
+
+Lastly, see the section on hazard rates in non-standard data sources below to determine when hazard rates may be 
+preferrable to annual incidence rates as a data source for cause model transition rates.
```

Comment:
Perhaps my confusion about the formula for incidence/remission probabilities below is related to the distinction between hazards and annual rates alluded to here.  If so, maybe there is a way to introduce and contrast these concepts up front to help me and other readers understand sooner.